src/MSALWrapper/AuthFlow/BadAuthFlowImplementException.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.MSALWrapper.AuthFlow
+{
+    using System;
+
+    /// <summary>
+    /// The bad authflow implementation exception.

```

Comment:
I think we could use a more descriptive summary here. What does this exception signify?

Reply:
Added more details, which should help.